pkgs/tools/misc/yajsv/default.nix

```
@@ -1,24 +1,22 @@
 { buildGoModule, fetchFromGitHub, lib }:
 
-let version = "1.4.0";
-in buildGoModule {
+buildGoModule rec {
   pname = "yajsv";
-  version = version;
+  version = "1.4.1";
 
   src = fetchFromGitHub {
     owner = "neilpa";
     repo = "yajsv";
     rev = "v${version}";
-    sha256 = "0smaij3905fqgcjmnfs58r6silhp3hyv7ccshk7n13fmllmsm7v7";
+    sha256 = "sha256-dp7PBN8yR+gPPUWA+ug11dUN7slU6CJAojuxt5eNTxA=";
```

Not particularly important but when using the "sha256-" type hashes, then I think it is a bit nicer to use the `hash` attribute:

```suggestion
    hash = "sha256-dp7PBN8yR+gPPUWA+ug11dUN7slU6CJAojuxt5eNTxA=";
```